_blog.yml

```
@@ -3099,3 +3099,13 @@
     - lcm
     - stable diffusion
     - guide
+
+- local: setfit-absa
+  title: "SetFitABSA: Few-Shot Aspect Based Sentiment Analysis using SetFit"
+  author: ronenlap
+  guest: true
+  thumbnail: /blog/assets/setfit-absa/intel_hf_logo.png
+  date: November XXX, 2023
```

```suggestion
  date: November 28, 2023
```

Just a reminder to update :)

---

Done :) Updated it to today, as I intend to release this later today